### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by genya0407**
`DataFrame#vectors=` checks if the argument is a `Daru::Index`.
However, because of a typo, it hasn't worked.

### CODE REVIEW

File: lib/daru/dataframe.rb [hunk 1]
```diff
@@ -1346,7 +1346,7 @@ def index= idx
     #   df.vectors = Daru::Index.new([:foo, :bar, :baz])
     #   df.vectors.to_a #=> [:foo, :bar, :baz]
     def vectors= idx
-      unless index.is_a?(Daru::Index)
+      unless idx.is_a?(Daru::Index)
```
Comment (v0dro) [inline lib/daru/dataframe.rb]: The original version is correct. Its checking if the argument `idx` is not of type `Daru::Index`.

Comment (genya0407) [inline lib/daru/dataframe.rb]: Do you mean that `idx` should be `Daru::Index` ?

Comment (v0dro) [inline lib/daru/dataframe.rb]: The above code is invalid. The RHS must be a `Daru::Index`.

I don't get this... the statement is `unless idx.is_a?(Daru::Index)`. That means that the error should be thrown iff the object passed is anything but a `Daru::Index`.

Comment (genya0407) [inline lib/daru/dataframe.rb]: Please let me confirm:

The following code is **valid**:
```ruby
df.vectors = Daru::Index.new [:a, :b, :c]
```

And, the following code is **invalid** and should **throw error**:

```ruby
df.vectors = [:a, :b, :c]
```
Is this ok? Am I correct?

In addition, let me confirm one more thing.
The original code is `index.is_a?(Daru::Index)`,
and the code I wrote is `idx.is_a?(Daru::Index)`.

Comment (v0dro) [inline lib/daru/dataframe.rb]: You're correct. My bad.

**Pull request merged**